### PR TITLE
Just all of my changes yet

### PR DESCRIPTION
**Pull request by lukeIam**

### CODE REVIEW

File: _data/contribute.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
 {
 	"title": "Report bugs",
 	"icon": "bug",
-	"description": "Help us keep Vocaluxe stable. If you find a bug, just open an issue in our GitHub issue tracker.",
+	"description": "Help us to keep Vocaluxe stable. If you find a bug, just open an issue in our GitHub issue tracker.",
```
Comment (derivator) [inline _data/contribute.json]: I had changed this [on purpose](https://english.stackexchange.com/questions/3578/what-is-the-correct-way-to-use-infinitive-after-the-verb-help-with-or-without) :D

File: _data/features.json [hunk 2]
```diff
@@ -19,19 +19,24 @@
 	"description": "Tired of re-entering the same names again and again? Create player profiles with pictures, so you'll always know at a glance who's supposed to be singing, and your high-score lists will always be correct."
 },
 {
-	"title": "Webcam and Wiimote",
-	"image": "",
-	"description": "Use your webcam to take pictures for the player profiles. Control Vocaluxe from the couch with your Wiimote."
+	"title": "Webcam",
+	"image": "webcam.png",
+	"description": "See yourself singing is even more funny - Vocaluxe supports webcams. Use it also to take pictures for the player profiles."
```
Comment (derivator) [inline _data/features.json]: > See yourself singing is even more funny

This sentence sounds strange to me, how about "Singing is even more fun when you can see yourself" ? "funny" is "witzig", not "spaßig", unless that's what you were going for. 

>Use it also to take pictures for the player profiles.

How about "You can also use it to take pictures for the player profiles." ?

File: _data/contribute.json (hunk 1 above)
Comment (lukeIam) [inline _data/contribute.json]: the whole change was not based on your PR but on the branch of flokuep (and I haven't seen this)

File: _data/features.json (hunk 2 above)
Comment (lukeIam) [inline _data/features.json]: > "Singing is even more fun when you can see yourself"  
> "You can also use it to take pictures for the player profiles."

Sounds better 👍

File: songs.html [hunk 3]
```diff
@@ -3,4 +3,4 @@
 section-type: songs
 title: Songs
 ---
-Playing Vocaluxe can be a lot of fun, but first you need some nice songs to sing...
+Playing Vocaluxe can be a lot of fun, but first you need some nice songs to sing...
```
Comment (derivator) [inline songs.html]: I guess it would be nice if this could be center-aligned?

**Pull request merged**